libnavigation-base/src/main/java/com/mapbox/navigation/base/options/RoutingTilesOptions.kt

```diff
@@ -16,6 +18,8 @@ import java.net.URI
  * One of (driving|driving-traffic|walking|cycling|truck).
  * @param tilesVersion version of tiles, chosen automatically if empty
  * @param filePath used for storing road network tiles
+ * @param tileStore tile store instance. It manages downloads and storage for requests to
```

Comment:
MIght be worth mentioning that for offline predictive caching use-cases this instance should be the same that is passed to map resource options.

Reply:
done